hash/psx.xml

```diff
@@ -260,8 +260,9 @@ For an overview of US PS1 discs that have not been dumped and added to redump's
 		</part>
 	</software>
 
-	<software name="2xtreme">
-		<!-- Original images (Redump)
+	<software name="2xtreme_us" cloneof="2xtreme">
+		<!--
+		http://redump.org/disc/2094/
 		<rom name="2Xtreme (USA).cue" size="700" crc="7e95393e" sha1="ed526fe2905f428530cc001038dd72c413bbe291"/>
 		<rom name="2Xtreme (USA) (Track 1).bin" size="45715824" crc="aad124a6" sha1="3f25e5d1366e60e70a416109711e109d48c9f19b"/>
 		<rom name="2Xtreme (USA) (Track 2).bin" size="56064624" crc="6f8d371b" sha1="9b986d87e8314c72757f629f6d7201bdeb0190b0"/>
```

Comment:
Let's open a discussion about this:
as part of their clearance http://redump.org/ gives this information already in the given URL. I legit don't think parsing XML comments as a pseudo-field is trivial, plus the verbosity of these track lines that don't even report back the msf to begin with, so it needs a roundtrip to the website anyway.
What if we just use an `<info name="redump.org" value="2094" />`, where of course 2094 is the ID that needs parsing thru clients including MAME UI itself?

Reply:
That's fair enough for me.
All the hash information is irrelevant, with the source and ID indicated.